tests/test_experiment_rsmsummarize.py

```diff
@@ -119,3 +120,18 @@ def test_run_experiment_summary_no_json():
                        source,
                        'rsmsummarize.json')
     do_run_summary(source, config_file)
+
+
+@raises(FileNotFoundError)
```

Comment:
Why does this test raise a `FileNotfoundError` if the test name ends with "_too_many_jsons()`?

Reply:
Should be ValueError